### PR TITLE
Custom styles for file Input

### PR DESCRIPTION
**Pull request by kamilszmajdzinski**
## PR Checklist

Please check all that apply to this PR using "x":

- [x] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [x] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

## Fixes

Issue Number: #138 

## What is the current behavior?
File input has default browser styling

## What is the new behavior?
File input has custom styling

### CODE REVIEW

File: examples/demo.css [hunk 1]
```diff
@@ -38,74 +43,114 @@ body, #app, .villain-demo {
...
+}
+
+.villain-demo .field .custom-file-input {
+  color: transparent;
+  border: none;
+  height: 26px;
+  padding: 0;
+  display: flex;
+}
+.villain-demo .field .custom-file-input::-webkit-file-upload-button {
```
Comment (btzr-io) [inline examples/demo.css]: This styles should work for all possible browsers

Comment (btzr-io) [reply examples/demo.css]: Also please post a screen preview

**Comment by btzr-io** [comment]
@kamilszmajdzinski are you still working on this ?
Please just open a pr at a time.

**Comment by btzr-io** [comment]
I'll close this for now , please let me know if you are still working on this :v:

**Comment by kamilszmajdzinski** [comment]
@btzr-io sorry for lack of response, I was pretty busy 😔, I am working on this now.

**Comment by btzr-io** [comment]
@kamilszmajdzinski Closing for inactivity (this requires simple changes so it should not take that long ), please, open a new pull request or feel free to check other issues: https://github.com/btzr-io/Villain/issues

**Pull request closed without merging**